### ISSUE TITLE
Install TRMM python version on Mac and linux

### ISSUE THREAD

**Issue by silversword411** [amidaware/tacticalrmm#1470]
Currently TRMM is only installing the Python3.8 on windows agent installs from https://github.com/amidaware/rmmagent/releases/download/v2.0.0/py38-x64.zip

Mac and linux agents assume python is on system already and doesn't install anything.

For the Community Scripts python module manager we don't want to accidentally install packages on the system installed (or auto-discovered) python from `env`

Add

- x64 for mac 
- x64 for linux
- x32 for other (Pi etc)

Basically, looking for a consistent base python version (TRMM managed) on any agent install we can install modules into and program against for script library stuff (OS agnostic scripts - They run everywhere!!!!).

**Comment by NiceGuyIT** [comment]
This might be possible with [PyOxy](https://pyoxidizer.readthedocs.io/en/stable/pyoxy.html).
